### ISSUE TITLE
Stress test?

### ISSUE THREAD

**Issue by tiagofiorini** [lpereira/hardinfo#568]
Hi,
Great software!
Could stress test be a feature?
I would suggest the Stress package, which is very convenient and easy to use.
The command "stress --cpu 4 --vm 4 --vm-bytes 1024M" would impose stress in 4 CPUs and 4Gb of RAM.
Together with temperature measurements of the CPU would be awesome!

**Comment by mckaygerhard** [comment]
interesting idea.. but seems hardinfo are used on winbuntu and other "easy to use linux"! so maybe this are more a expert usage right?

**Comment by lpereira** [comment]
@tiagofiorini Yes, it absolutely could be a feature.  Other similar tools for Windows do exist and HardInfo could provide it.  I'll think about this, but please don't hold your breath. :)

**Comment by tiagofiorini** [comment]
@mckaygerhard yes, my point is that hardinfo could allow some predefined stress tests thus an user with or without experience could perform such a test conveniently... even glxgears or glmark2 could provide GPU stress tests disabling vertical syncing. I can offer some limited help since I am not an expert neither in coding nor in hardware test. But I know how to run some simple tests using these tools.
